aspnetcore/web-api/http-repl-tool.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Use the HTTP REPL tool
+author: scottaddie
+description: Learn how to use the HTTP REPL .NET Core Global Tool to browse and test an ASP.NET Core web API.
+monikerRange: '>= aspnetcore-3.0'
+ms.author: scaddie
+ms.custom: mvc
+ms.date: 06/21/2019
+uid: web-api/http-repl-tool
+---
+# Use the HTTP REPL tool
+
+The HTTP REPL is a cross-platform tool that's supported everywhere .NET Core is supported.

```

Comment:
Some more detail is needed. I suggest adding another sentence or two explaining the tool's purpose. Maybe also mention that it's a command-line alternative to tools like Fiddler and Postman.

Reply:
I didn't notice that earlier. That the description was not applicable. I added more details. I don't want to mention postman or fiddler as I don't want to get into the communalities and the differences. This might be better suited for a blog post.